### DISCUSSION TITLE
Price check - offline default option

### DISCUSSION THREAD

**Discussion by StochasticBoris** [SnosMe/awakened-poe-trade#1414] [Ideas]
When price checking in markets with volumes, users often expand to offline results for a better sample size. Currently, for each checked item, a search request must be performed (online only default), only then can you change the results to include offline as well. This adds to a lot of extra API calls and UI clicking over time.

I would love it if there were a checkbox in settings, which would by default add offline items postings to results when price checking an item.

**Comment by sellthisonyagit-cell** [comment]
Additionally allow us to preset / remember other toggles, such as "Merchant only"
